lambdas/indexer/app.py

```diff
@@ -45,6 +48,16 @@ def __init__(self):
                          # see LocalAppTestCase.setUpClass()
                          unit_test=globals().get('unit_test', False))
 
+    @property
+    def current_auth_token(self) -> Optional[str]:
+        auth_token = super().current_auth_token
+        if auth_token is None:
+            return None
+        elif auth_token.lower().startswith('bearer: '):
```

Comment:
Is there a way to check for an HMAC auth header, rather than disallowing only one of the known unsupported ones? In security, you always want to disallow by default and only allow a specific set of supported cases. If there isn't, that should be explained in a comment.